docs/src/dataDisplay/dataCards/examples/exampleDataCards.jsx

```diff
@@ -0,0 +1,48 @@
+import {
+    DataCards,
+} from 'react-cm-ui';
+import React from 'react';
+
+const persons = [
+    {
+        name: 'John',
+        age: 22,
+        gender: 'male',
+    },
+    {
+        name: 'John',
+        age: 33,
+        gender: 'male',
+    },
+    {
+        name: 'John',
+        age: 44,
+        gender: 'male',
+    },
+];
+
+function ExampleDataCards() {
+    return (
+        <DataCards
+            cardProps={() => ({
+                // eslint-disable-next-line no-console
+                onClick: () => console.log('Click!'),
```

Comment:
Should be `onClick: () => { console.log('Click!'); },`

Reply:
Fixed.

Reply:
Looks good.  Resolving.